### ISSUE TITLE
signTypedData_v4 does not overflow:wrap fields.

### ISSUE THREAD

**Issue by danfinlay** [MetaMask/metamask-extension#7921]
Much of the point of `signTypedData` is that its fields are human-readable, and so as a user hoping to approve meta-transactions using this method, I should be able to read the entire field.

This is how it looks today:

<img width="472" alt="Screen Shot 2020-01-28 at 3 34 14 PM" src="https://user-images.githubusercontent.com/542863/73314981-b21f5180-41e3-11ea-9b14-4e4fe036cc95.png">

A couple issues:
- [ ] Fields overflow to be unreadable
- [ ] The title `Message` appears twice

**Comment by BboyAkers** [comment]
I'll take a crack at this one 🙂

**Comment by BboyAkers** [comment]
Pushed a fix 🙂,  @Gudahtt let me know your thoughts!
